hack/make/tests/e2e.mk

```diff
@@ -64,20 +64,28 @@ test/e2e/cloudnative/codemodules: manifests/crd/helm
 
 ## Runs CloudNative codemodules-with-proxy e2e test only
 test/e2e/cloudnative/codemodules-with-proxy: manifests/crd/helm
-	go test -v -tags "$(shell ./hack/build/create_go_build_tags.sh true)" -timeout 20m -count=1  ./test/scenarios/istio -args --feature "codemodules-with-proxy" $(SKIPCLEANUP)
+	go test -v -tags "$(shell ./hack/build/create_go_build_tags.sh true)" -timeout 20m -count=1  ./test/scenarios/istio -args --feature "codemodules-with-proxy$$" $(SKIPCLEANUP)
```

Comment:
why the `$$`

Reply:
[see above ](https://github.com/Dynatrace/dynatrace-operator/pull/4651#discussion_r2015916837)